oprf/oprf_test.go

```diff
@@ -0,0 +1,150 @@
+package oprf_test
```

Comment:
Can we add tests against the test vectors?

Reply:
I'll change the sage code first.

Reply:
The were just added. There is some difference, that I'll explain in a call.